docs/resources/confluent_flink_artifact.md

```diff
@@ -38,10 +37,9 @@ resource "confluent_flink_artifact" "main" {
 
 The following arguments are supported:
 
-- `display_name` - (Required String) The display name of Flink Artifact.
+- `display_name` - (Required String) The unique name of the Flink Artifact per cloud, region, environment scope.
 - `cloud` - (Required String) The cloud service provider that runs the Flink Artifact.
 - `region` - (Required String) The cloud service provider region that hosts the Flink Artifact.
-- `class` - (Required String) Java class or alias for the Flink Artifact as provided by developer.
```

Comment:
Let's not remove it outright, but rather mark it as deprecated.

```suggestion
- `class` - (Required String, , **Deprecated**) Java class or alias for the Flink Artifact as provided by developer.
```

Reply:
also I don't have a full context here, but do we need to mark it as optional now?